src/DssSpell.t.sol

```diff
@@ -220,10 +215,10 @@ contract DssSpellTest is DSTest, DSMath {
         // Test for all collateral based changes here
         //
         afterSpell.collaterals["ETH-A"] = CollateralValues({
-            aL_enabled:   false,           // DssAutoLine is enabled?
-            aL_line:      0 * MILLION,     // In whole Dai units
-            aL_gap:       0 * MILLION,     // In whole Dai units
-            aL_ttl:       0,               // In seconds
+            aL_enabled:   true,            // DssAutoLine is enabled?
+            aL_line:      1500 * MILLION,  // In whole Dai units
+            aL_gap:       30 * MILLION,    // In whole Dai units
+            aL_ttl:       12 hours,        // In seconds
             line:         1000 * MILLION,  // In whole Dai units
```

Comment:
Maybe set it to `0` with the same comment we have in `ETH-B`

Reply:
yup, `0 * MILLION`

Reply:
I think maybe next time we can modify this so that `line` is shared and we don't have the redundancy her. If auto_line is on, line is the max available, otherwise it is the expected line.

Reply:
sounds good to me